### PR TITLE
Use stylelint-config-upleveled standalone package

### PR DESCRIPTION
**Pull request by Josehower**
close https://github.com/upleveled/eslint-config-upleveled/issues/218

since now we have the [Stylelint config as a standalone package](https://www.npmjs.com/package/stylelint-config-upleveled) we want to use the package instead of the manual installation we have been doing.

This PR updates the script to install the standalone package and simplifies the configuration template for Stylelint

### CODE REVIEW

**Comment by karlhorky** [comment]
`__tests__` are still not finished yet - search the codebase to make sure that all Stylelint things are upgraded

File: __tests__/package.json [hunk 1]
```diff
@@ -1,28 +1,20 @@
...
   },
   "devDependencies": {
-    "@ts-safeql/eslint-plugin": "^0.0.28",
-    "eslint-config-upleveled": "^4.0.12",
-    "libpg-query": "^15.0.2",
-    "postcss-styled-syntax": "^0.4.0",
-    "stylelint": "^15.4.0",
-    "stylelint-config-css-modules": "^4.2.0",
-    "stylelint-config-recommended": "^11.0.0",
-    "stylelint-config-recommended-scss": "^10.0.0",
```
Comment (Josehower) [inline __tests__/package.json:22]: Removing libraries that are now in stylelint package

File: __tests__/stylelint.config.cjs [hunk 2]
```diff
@@ -1,40 +1,6 @@
...
-        '**/*.js',
-        '**/*.cjs',
-        '**/*.mjs',
-        '**/*.jsx',
-        '**/*.ts',
-        '**/*.tsx',
-      ],
-      customSyntax: 'postcss-styled-syntax',
-    },
-  ],
```
Comment (Josehower) [inline __tests__/stylelint.config.cjs:37]: use the config from the new package

File: bin/install.js [hunk 3]
```diff
@@ -49,13 +49,7 @@ if (
 }
 
 if ('react-scripts' in projectDependencies || 'next' in projectDependencies) {
-  newDevDependenciesToInstall.push(
-    'postcss-styled-syntax',
-    'stylelint',
-    'stylelint-config-css-modules',
-    'stylelint-config-recommended',
-    'stylelint-config-recommended-scss',
-  );
```
Comment (Josehower) [inline bin/install.js:58]: this dependencies are now added in the installation of stylelint-config-upleveled

File: templates/stylelint.config.cjs [hunk 4]
```diff
@@ -1,40 +1,6 @@
...
-        '**/*.js',
-        '**/*.cjs',
-        '**/*.mjs',
-        '**/*.jsx',
-        '**/*.ts',
-        '**/*.tsx',
-      ],
-      customSyntax: 'postcss-styled-syntax',
-    },
-  ],
```
Comment (Josehower) [inline templates/stylelint.config.cjs:37]: simplify config similar to https://github.com/upleveled/eslint-config-upleveled/pull/220/files#r1166967732

File: __tests__/package.json [hunk 5]
```diff
@@ -1,28 +1,20 @@
 {
   "name": "eslint-config-upleveled-tests",
   "version": "1.0.0",
-  "main": "index.js",
-  "author": "Karl Horky <karl.horky@gmail.com>",
-  "license": "MIT",
   "private": true,
+  "license": "MIT",
+  "author": "Karl Horky <karl.horky@gmail.com>",
+  "main": "index.js",
```
Comment (karlhorky) [inline __tests__/package.json:7]: if these lines will be re-sorted, you can also change the author to whatever we normally write for UpLeveled and add a `contributors` array, like mentioned in the Stylelint config PR:

- https://github.com/upleveled/stylelint-config-upleveled/pull/3

File: __tests__/package.json [hunk 6]
```diff
@@ -1,28 +1,22 @@
...
-    "postcss-styled-syntax": "^0.4.0",
-    "stylelint": "^15.4.0",
-    "stylelint-config-css-modules": "^4.2.0",
-    "stylelint-config-recommended": "^11.0.0",
-    "stylelint-config-recommended-scss": "^10.0.0",
+    "@ts-safeql/eslint-plugin": "0.0.28",
+    "eslint-config-upleveled": "4.0.13",
+    "libpg-query": "15.0.2",
+    "postcss-styled-syntax": "0.4.0",
+    "stylelint-config-upleveled": "1.0.2",
```
Comment (Josehower) [inline __tests__/package.json:19]: Add dependencies that should be installed with pnpm add --save-dev eslint-config-upleveled@latest that are necessary to run the tests inside of this directory

Comment (Josehower) [reply __tests__/package.json:24]: https://github.com/upleveled/eslint-config-upleveled/pull/220/commits/2bf8840f52e0b13dd127f65d00fc147a4c5c5452
https://github.com/upleveled/eslint-config-upleveled/pull/220/commits/690a06c9694734491dca25dcd2d22f6b39f76e45

File: __tests__/package.json (hunk 5 above)
Comment (Josehower) [reply __tests__/package.json]: https://github.com/upleveled/eslint-config-upleveled/pull/220/commits/2bf8840f52e0b13dd127f65d00fc147a4c5c5452
https://github.com/upleveled/eslint-config-upleveled/pull/220/commits/690a06c9694734491dca25dcd2d22f6b39f76e45

**Review by karlhorky — APPROVED** [review]

**Pull request merged**